app/src/main/java/com/example/msogcardwar/Game_Activity.java

```diff
@@ -223,15 +235,19 @@ private void createGameStacks() {
         Collections.shuffle(all_card_stack);
 
         // Divide the cards between the players.
-        while(all_card_stack.size() != 0){
-            if(player_card_flag){
-                insertCardToPlayerOneStack(all_card_stack.get(0));
-                player_card_flag = false;
-            }else{
-                insertCardToPlayerTwoStack(all_card_stack.get(0));
-                player_card_flag = true;
-            }
+        // In case of tie, give only 5 cards.
+        while(all_card_stack.size() != 0 && i < 5){
+            // Add card to player one stack and remove it from overall stack
```

Comment:
// Add card to player one stack and remove it from the overall stack
Cange the overall stack --> list